lib/stache/mustache/view/method_missing_view_arity.rb

```diff
@@ -0,0 +1,30 @@
+class Stache::Mustache::View
+  protected
+
+  def method_missing(method, *args, &block)
+    if view_method_arity_matches?(method, *args)
+      view.send(method, *args, &block)
+    else
+      super
+    end
+  end
+
+  ##
+  # Do a crude sanity check on number of parameters for method delegation to view
+  def view_method_arity_matches?(method, *args)
```

Comment:
Assignment Branch Condition size for view_method_arity_matches? is too high. [23.69/15]<br>Method has too many lines. [12/10]